lib/ansible/modules/cloud/ovirt/ovirt_auth.py

```diff
@@ -221,17 +238,23 @@ def main():
     elif state == 'absent':
         params = module.params['ovirt_auth']
 
+    url = params.get('url') or os.environ.get('OVIRT_URL')
```

Comment:
```
url = params.get('url') or os.environ.get('OVIRT_URL')
if not url: module.fail_json('url is a required parameter[..]')
``` 
Some of these parameters were checked in `required_if` but now also allow environment variables, so if neither one yields a result, you need to fail out with a message telling them what was missing.